test/general/packet.js

```diff
@@ -600,6 +600,115 @@ module.exports = () => describe('Packet', function() {
     });
```

Comment:
We should add a test that generates a v6 signed message and confirms that the OPS is the correct version and it is used to verify

Reply:
Added a test that checks if a one-pass signature packet is correctly generated when (inline) signing with a v6 key.